src/todo.js

```diff
@@ -0,0 +1,119 @@
+import { reset } from "ansi-colors";
+
+let tasks = JSON.parse(localStorage.getItem('tasks')) || [];
+
+export function renderTasks() {
+  const taskList = document.getElementById('task-list');
+  taskList.innerHTML = '';
+
+  tasks.forEach((task, index) => {
+    const listItem = document.createElement('li');
+    listItem.draggable = true; // Enable dragging for each task
+
+    const checkBox = document.createElement('input');
+    checkBox.type = 'checkbox';
+    checkBox.checked = task.completed;
+    checkBox.dataset.index = index;
+    listItem.appendChild(checkBox);
+
+    const taskDescription = document.createElement('span');
+    taskDescription.textContent = task.description;
+    taskDescription.classList.add('task-description');
+    if (task.completed) {
+      taskDescription.classList.add('completed');
+    }
+    listItem.appendChild(taskDescription);
+
+    const editButton = document.createElement('button');
+    editButton.textContent = '';
+    editButton.className = 'edit-btn';
+    editButton.dataset.index = index;
+    listItem.appendChild(editButton);
+
+    const editIcon = document.createElement('i');
+    editIcon.className = 'fas fa-ellipsis-v';
+    editButton.appendChild(editIcon);
```

Comment:
You did well with your project, kindly note that the function for editing a task isn't working well and throws an error before it is edited. Please try to fix it to ensure full functionality of the app 😄 

![image](https://github.com/abdullah97-pro/ToDo-list-add-remove/assets/20537712/3b821135-130e-46b0-b34c-21f86dbcf88e)